tutorbackup/patches/k8s-jobs

```diff
@@ -31,6 +31,10 @@ spec:
               value: '{{ MYSQL_ROOT_USERNAME }}'
             - name: MYSQL_ROOT_PASSWORD
               value: '{{ MYSQL_ROOT_PASSWORD }}'
+            {% if BACKUP_MYSQL_DATABASES %}
+            - name: MYSQL_DATABASES
+              value: '{{ BACKUP_MYSQL_DATABASES | join(" ") }}'
```

Comment:
The single quotes here should also be removed.

Reply:
done

Reply:
It's still there though.